backend/src/controllers/DailyWorkoutsController.js

```diff
@@ -24,7 +28,89 @@ const getDailyWorkoutsbyDate = async (req, res) => {
     }
 }
 
+
+  const insertWorkout = async (req, res) => {
+    const userId = req.user.data.user_id;
+    const { dailyworkout_id, workout_id, sets, reps } = req.body;
+    const currentDate = new Date(); // Get the current date 
+    console.log("hit");
+    try {
+        const workoutExists = await db.oneOrNone(
+            "SELECT * FROM dailyworkouts WHERE dailyworkout_id = $1",
+            [dailyworkout_id]
+        );
+
+        if (!workoutExists) {
+            const insertedWorkout = await db.one(
+                "INSERT INTO dailyworkouts(dailyworkout_id, user_id, workout_id, sets, reps, date) VALUES($1, $2, $3, $4, $5, $6) RETURNING *",
+                [dailyworkout_id, userId, workout_id, sets, reps, currentDate]
+            );
+            return res.status(200).json({ success: true, workout: insertedWorkout, message: "Workout inserted successfully" });
+        } else {
+            return res.status(400).json({ success: false, message: "Workout already exists" });
+        }
```

Comment:
This logic is slightly inverted. We should only return a status code `200` if a workout exists. Not the other way around. Return `400` if a workout **_does not_** exist.

Take a look at the design below:
![image](https://github.com/KyoshiNoda/Elevate/assets/62672803/a4ca7296-c4d9-45d2-ac20-38788a54f5ca)